src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -26,6 +26,7 @@
               :title="step.title"
               :editable="index + 1 < wizardStore.step && wizardStore.listComponentMode !== 'add'"
               :complete="index + 1 < wizardStore.step"
+              style="white-space: nowrap; font-size: 0.9em"
```

Comment:
Hey just for consistency can you please replace `nowrap` with:

`class="text-no-wrap"`

Which font size are you going for here? Can we use one of our classes from `typography.scss`?

Reply:
Thanks for the review. addressed both concerns, using this now:
 :class="`small ${mdAndDown ? 'text-wrap' : 'text-no-wrap'}`"